### PR TITLE
Peripona delhi --- got Some Milk ???

### LINKED ISSUE #27: Peripona Recommendation for Delhi, India
# Peripon Delhi India
- [ ] Create a branch
- [ ] Add a File
- [ ] Commit my chages
- [ ] Open a Pull Request 
- [ ] Collaborate with my new Friends (YOU)
- [ ] Get a 👍 
- [ ] Merge and Pull Request to Master

### PR DESCRIPTION
**Pull request by Peripona**
This PR closes #27 By adding 

That'a alot of typing ok.. 

@stoe  @UTCWebDev @halfaro1  That's it I can't do any better than this ever 💃

### CODE REVIEW

File: peripona-delhiindia.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Delhi-India
+
```
Comment (Peripona) [inline peripona-delhiindia.md]: Add some thing here..

**Comment by UTCWebDev** [comment]
@Peripona 👋 thanks for starting. 
Do you know anyone in Pune? 
I collaborated with some Pune developers on a project.
- Mithul Maitri
- Samir Deshpande

**Comment by passion4aix** [comment]
:+1:

**Comment by passion4aix** [comment]
:+1:

**Comment by UTCWebDev** [comment]
http://iceland.trawire.com

**Pull request merged**